apps/jobs/views.py

```diff
@@ -350,3 +351,31 @@ def get_remaining_submissions(request, challenge_phase_pk, challenge_pk):
                          'remaining_submissions': remaining_submission_count
                          }
         return Response(response_data, status=status.HTTP_200_OK)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_submission_by_pk(request, submission_id):
+    """
+    Returns the details of a single submission.
```

Comment:
Please update the comments with this.
**API endpoint to fetch the details of a submission. Only the submission owner or the challenge hosts are allowed.**